test/distributed/org/apache/cassandra/distributed/shared/ClusterUtils.java

```diff
@@ -611,8 +632,8 @@ public static void waitForCMSToQuiesce(ICluster<IInvokableInstance> cluster, Epo
                 if (cluster.get(j).isShutdown())
                     continue;
                 Epoch version = getClusterMetadataVersion(cluster.get(j));
-                if (!awaitedEpoch.equals(version))
-                    notMatching.add(new ClusterMetadataVersion(j, version, getClusterMetadataVersion(cluster.get(j))));
+                if (!awaitedEpoch.equals(version) && version.getEpoch() < awaitedEpoch.getEpoch())
```

Comment:
not sure if this breaks anything on your end, but I found that I would see epoch=33 then fail because every node is on epoch=34... so I use this more as a min wait barrier than anything else...